runtime/onert/backend/trix/ops/BulkLayer.cc

```diff
@@ -49,24 +50,57 @@ void BulkLayer::configure(const std::vector<const IPortableTensor *> &inputs,
     throw std::runtime_error("Unable to extract the model metadata");
   }
 
+  _model_id.resize(_dev_context->getDevSize());
+
   generic_buffer model_file;
   model_file.type = BUFFER_FILE;
   model_file.filepath = binary_path.c_str();
   model_file.size = _meta->size;
 
-  if (registerNPUmodel(dev_context->getDev(), &model_file, &_model_id) < 0)
+  if (registerNPUmodel(dev_context->getDev(0), &model_file, &_model_id[0]) < 0)
+  {
+    throw std::runtime_error("Failed to register npu model");
+  }
+  if (registerNPUmodel(dev_context->getDev(1), &model_file, &_model_id[1]) < 0)
```

Comment:
It seems to assume device number is 2. It would be better to use iterate with number_of_devices. As I understand, it is `2` for both x86 and arm. It is also preferred to introduce constant instead of magic number `2`.

Reply:
I remove the usage of magic number 2